### PR TITLE
Create list of smart contract addresses

### PR DESCRIPTION
**Pull request by rcai1**

### CODE REVIEW

File: documentation/getting_started/developer_reference/contract_addresses.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Deployed DVM Contracts
+Below is where you can find smart contract addresses for UMA-supported mainnet and testnet deployments of the DVM. 
+
+## UMA Token Holders
+If you are a UMA token holder, you will probably only interact with `Voting`, `Finder`, and `DesignatedVotingFactory`.
```
Comment (mrice32) [inline documentation/getting_started/developer_reference/contract_addresses.md:5]: I would add the Governor to this list.

Comment (mrice32) [reply documentation/getting_started/developer_reference/contract_addresses.md:5]: Sorry if I gave you conflicting feedback before.

Comment (rcai1) [reply documentation/getting_started/developer_reference/contract_addresses.md:5]: Sounds good, I'll promote it to the same line here.

Comment (mrice32) [reply documentation/getting_started/developer_reference/contract_addresses.md]: Oh, man, I really didn't read that carefully. I'm sorry. Didn't realize you had an entire line about the governor below. Sorry about that :/

Comment (rcai1) [reply documentation/getting_started/developer_reference/contract_addresses.md]: No problem - don't think it's a big deal!

**Pull request merged**